packages/design-system/src/components/Table/Table.jsx

```diff
@@ -83,7 +83,7 @@ export class Table extends React.PureComponent {
             _scrollableNotice: this.props.scrollableNotice,
           });
         }
-      } else if (this.props.stackable) {
+      } else if (this.props.stackable && child.props) {
```

Comment:
Why are we adding `child.props` here?

Reply:
We want to avoid extending to text/strings and only extend if child is an object.